lib/taskana-core/src/main/java/pro/taskana/task/api/TaskState.java

```diff
@@ -4,5 +4,20 @@
 public enum TaskState {
   READY,
   CLAIMED,
-  COMPLETED
+  COMPLETED,
+  CANCELLED,
+  TERMINATED;
+
+  public boolean isInStates(TaskState... states) {
```

Comment:
Maybe just "in" as it is a well known operator for this? Just an idea.

Reply:
You don't like that? Or you has overseen this ons?

Reply:
Ok, I didn't see this one. No problem, I'll change it to 'in'